modules/apps/frontend-js/frontend-js-web/test/liferay/delegate/delegate.es.js

```diff
@@ -181,4 +181,61 @@ describe('delegate', () => {
 
 		expect(delegateTarget).toBe(document.querySelector('.match'));
 	});
+
+	it('works with multiple selectors', () => {
```

Comment:
Just for the sake of precision, can we reword this to?

```js
it('works with comma-delimited selector lists', () => {
```

(because "selector list" is the technical name for this thing)

Reply:
of course